proto/hit_tracker.proto

```diff
@@ -0,0 +1,42 @@
+
+syntax = "proto3";
+
+import "google/protobuf/duration.proto";
+import "proto/resource.proto";
+
+package hit_tracker;
+
+// Contains information about an artifact that was downloaded from the cache.
+message Download {
```

Comment:
just sanity checking - are downloads the only thing that need to be forwarded to upstream? (I guess everything else makes an upstream request, so it's fine for the app to track?)

also, just dropping a note/reminder that we might need to consider what happens with the hit_tracker if we're still planning to allow the cache proxy to serve the AC result payloads.